src/value/owned.rs

```diff
@@ -6,6 +6,9 @@
 //! ownership of data to do so.
 
 use super::{AnyInt, Element, ImportSource, Sequence, Struct, SymbolToken};
+use crate::result::IonError::IllegalOperation;
```

Comment:
There are some warnings here around "unused import", are those warnings valid?

Reply:
yes. I will remove it.